vms/platformvm/execconfig/execution_config.go

```diff
@@ -0,0 +1,51 @@
+package execconfig
+
+import (
+	"encoding/json"
+
+	"github.com/ava-labs/avalanchego/utils/units"
+)
+
+const (
+	blockCacheSize               = 64 * units.MiB
+	txCacheSize                  = 128 * units.MiB
+	transformedSubnetTxCacheSize = 4 * units.MiB
+
+	validatorDiffsCacheSize = 2048
+	rewardUTXOsCacheSize    = 2048
+	chainCacheSize          = 2048
+	chainDBCacheSize        = 2048
```

Comment:
[linter says this is unused](https://github.com/ava-labs/avalanchego/actions/runs/5427007633/jobs/9907808069?pr=1677), which makes static analysis fail. Please drop it

Reply:
it comes from `vms/platformvm/state/state.go:44:2` I'm going to delete them